package.json

```diff
@@ -73,7 +73,7 @@
     "@open-wc/dedupe-mixin": "^1",
     "focus-visible": "^5",
     "ifrau": "^0.40",
-    "intl-messageformat": "^7",
+    "intl-messageformat": "^10",
```

Comment:
Just because seeing this triggered a vague memory of "upgrading this will be a problem":
https://github.com/BrightspaceUI/core/issues/630

From that issue, the problems were Polymer CLI-related, which we no longer care about. 🎉